src/db/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Connection, ConnectionOptions, createConnection } from "typeorm";
+import environment, { getOptionsFromEnvironment } from "./environment"
```

Comment:
```suggestion
import environments, { getOptionsFromEnvironment } from "./environment"
```

This way, you don't have to rename the variable before re-exporting